content/en/blog/_posts/2021-04-08-PodSecurityPolicy-Past-Present-and-Future.md

```diff
@@ -0,0 +1,74 @@
+---
+layout: blog
+title: "PodSecurityPolicy: Past, Present, and Future"
+date: 2021-04-08
+slug: PodSecurityPolicy-Past-Present-and-Future
+---
+
+**Author:** Tabitha Sable (Kubernetes SIG Security)
+
+PodSecurityPolicy (PSP) is being deprecated in Kubernetes 1.21. This starts the countdown to its removal, but doesn’t change anything else. PodSecurityPolicy will continue to be fully functional for several more releases before being removed completely. In the meantime, we are developing a replacement for PSP that covers key use cases more easily and sustainably.
```

Comment:
PodSecurityPolicy is _also_ the `kind` of an object in the API you use to configure that admission controller's behavior. Some people will only have seen the API end of things, not the kube-apiserver configuration that makes it work.